app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentSheetController.kt

```diff
@@ -119,9 +123,17 @@ class ScheduleAppointmentSheetController @Inject constructor(
   }
 
   private fun scheduleDefaultAppointmentDateForSheetCreates(events: Observable<UiEvent>): Observable<PotentialAppointmentDate> {
-    return events
-        .ofType<ScheduleAppointmentSheetCreated>()
-        .withLatestFrom(configProvider) { _, config -> config.defaultTimeToAppointment }
```

Comment:
If the `defaultTimeToAppointment` property is no longer necessary, we could remove this. However, we might want to retain it as a fallback for cases where users have not yet synced facilities after updating to this version (see the other comment).